lib/ansible/modules/cloud/atomic/atomic_image.py

```diff
@@ -87,9 +103,38 @@ def core(module):
     image = module.params['name']
     state = module.params['state']
     started = module.params['started']
+    backend = module.params['backend']
     is_upgraded = False
 
     module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C')
+    out = {}
+    err = {}
+    rc = 0
+
+    if backend:
+        if state == 'present' or state == 'latest':
+            args = ['atomic', 'pull', "--storage=%s" % backend, image]
+            rc, out, err = module.run_command(args, check_rc=False)
+            if rc < 0:
+                module.fail_json(rc=rc, msg=err)
+            else:
+                out_run = ""
+                if started:
+                    args = ['atomic', 'run', "--storage=%s" % backend, image]
```

Comment:
`changed` will be False.  `changed` is True only when a new image was pulled (either to Docker or to OSTree)